symengine/tests/basic/test_solve.cpp

```diff
@@ -375,3 +379,88 @@ TEST_CASE("solve_poly", "[Solve]")
     REQUIRE(eq(*soln, *finiteset({neg(one), neg(integer(2))})));
 #endif
 }
+
+TEST_CASE("linsolve", "[Solve]")
+{
+    auto x = symbol("x"), y = symbol("y"), z = symbol("z"), t = symbol("t");
+    // Augmented Matrix as input
+    auto aug = DenseMatrix(
+        4, 5, {integer(1), integer(2), integer(3), integer(4), integer(10),
+               integer(2), integer(2), integer(3), integer(4), integer(11),
+               integer(3), integer(3), integer(3), integer(4), integer(13),
+               integer(9), integer(8), integer(7), integer(6), integer(30)});
+    vec_basic solns = linsolve(aug, {x, y, z, t});
+    REQUIRE(solns.size() == 4);
+    REQUIRE(eq(*solns[0], *integer(1)));
+    REQUIRE(eq(*solns[1], *integer(1)));
+    REQUIRE(eq(*solns[2], *integer(1)));
+    REQUIRE(eq(*solns[3], *integer(1)));
+
+    // vector of equations as input
+    solns
+        = linsolve({add({x, mul(integer(2), y), mul(integer(3), z),
+                         mul(integer(4), t), integer(10)}),
```

Comment:
No.
Its a mistake from this [line](https://github.com/symengine/symengine/pull/1317/files#diff-b7853adba5c900a6a7c31d27aa474673R399)